doc/script_commands.txt

```diff
@@ -3129,6 +3129,106 @@ Valid types are:
 
 See the sample in 'doc/sample/getiteminfo.txt'.
 
+Returned values and their constants (constant = value),
+one can use either the value or the constants for your checks:
+
+	ITEMINFO_TYPE:
+		IT_HEALING 		= 0
+		IT_UNKNOWN 		= 1
+		IT_USABLE  		= 2
+		IT_ETC     		= 3
+		IT_ARMOR   		= 4
+		IT_WEAPON  		= 5
+		IT_CARD    		= 6
+		IT_PETEGG  		= 7
+		IT_PETARMOR 	= 8
+		IT_UNKNOWN2 	= 9
+		IT_AMMO     	= 10
+		IT_DELAYCONSUME = 11
+		IT_SHADOWGEAR	= 12
+		IT_CASH 		= 18
```

Comment:
```suggestion
		IT_HEALING
		IT_UNKNOWN
		IT_USABLE
		IT_ETC
		IT_ARMOR
		IT_WEAPON
		IT_CARD
		IT_PETEGG
		IT_PETARMOR
		IT_UNKNOWN2
		IT_AMMO
		IT_DELAYCONSUME
		IT_SHADOWGEAR
		IT_CASH
```

Reply:
I still feel like adding the value being returned from the command useful.
Imagine, you use the command and get 18 for Type, unless you look into the src you won't know what IT that is.
It would be different if the command itself would return the constant, but it doesn't.

Reply:
You should not look for any numbers.
You should implement your check based on what you look for.

After your update you will see the list of types here in the documentation.
And you will think oh I want to look for cards.
This means you copy IT_CARD and check for it in your "if".

If we ever decide to change the number (or have to change the number), it will screw up all your scripts otherwise.

Reply:
Alright, I will remove them, since I'm on it already I will take a safe measurement as well and remove these numbers as well:
![image](https://github.com/user-attachments/assets/50c8e65d-26ba-4f32-be0b-ac4973289e36)
And change the script to not use a loop.